integration-test/package.json

```diff
@@ -17,6 +17,9 @@
     "build:rollup": "rollup -c scaffolding/rollup/rollup.config.test.js",
     "build:parcel": "parcel build scaffolding/parcel/index.html --public-url ./ -d bundles/parcel"
   },
+  "browserslist": [
+    "cover 50%"
```

Comment:
Rationale (currently in a commit comment) should go into a source comment somewhere.

Reply:
And clearly we eventually need a better way to say what our platform baseline is, both for testing and production purposes.